docs/book/advanced-guide/gcp_entrypoint.md

```diff
@@ -0,0 +1,277 @@
+# Getting started with Google Cloud
+
+This guide will take you from the default local stack to an opinionated cloud 
+stack in no time. This should help you get started with running your machine 
+learning pipeline in production.
+
+
+{% hint style="info" %}
+This guide represents **one** of many ways to create a cloud stack on gcp. 
```

Comment:
```suggestion
This guide represents **one** of many ways to create a cloud stack on GCP. 
```

Reply:
I think this should probably be uppercase right? There are lot's of other occurrences below in case we decide to change it.